docs/technical_reference/unit_models/treatment_models/med_md_semibatch.rst

```diff
@@ -0,0 +1,142 @@
+
+For the Mixer and Separator unit model, please refer to the documentation of IDAES package.
+
+* Initial condition at the first period
+
+VAGMD feed temperature and concentration are the same as MED brine at the beginning of the processing progress, 
+and so is the mixed liquid in M1, assuming the initial volume in M1 is the user specified batch volume.
+The MED brine tank starts as empty.
+
+* Discharging and refilling phase 
+
+The length of discharging and refilling phase is determined by the following equation:
+
+:math:`T_{reflling} = \frac{V_{batch} - V_{tank_stored}}{\nu_{MED,brine} - \nu_{MD,distillate}}`
+
+where :math:`\nu_{MED,brine}` and :math:`\nu_{MD,distillate}` are the volumetric flow rate of MED brine and MD distillate
+
+* System scaling
+
+The system can be scaled for cost analysis purpose. The user needs to specify the target system capacity
+``target_system_capacity``, and the model will calculatesthe capacity of MED and MD respectively, based on the 
+performance of a specified semibatch operation.
+
+
+Methods
+-------
+
+This unit model class includes the following methods for functionalities. 
+
+``add_costing_packages``: Creates a costing block and setup the system based on the target capacity.
+
+``get_model_performance``: Returns the overall performance of the batch operation in a dictionary, and the timewise system performance of the processing phase in a pandas dataframe.
+
+``get_costing_performance``: Returns the cost performance of a system with a target capacity scaled upon the specified MED-MD module in the unit model.
+
+
+Equations
+---------
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Water production from MED during processing phase", ":math:`m_{prod,med,process} = {\sum_{t=1}^{N} m_{distillate,t}d_{t}}`"
+   "Water production from MD during processing phase", ":math:`m_{prod,md,process} = {\sum_{t=1}^{N} Pflux_{t} A_{module} d_{t}}`"
+   "Water production from MED during refilling phase", ":math:`m_{prod,med,refilling} =  m_{distillate,t=0} t_{reflling}`"
+   "Water production from MD during refilling phase", ":math:`m_{prod,md,refilling} =  Pflux_{t=0} A_{module} t_{reflling}`"
+   "Total water production during one batch", ":math:`m_{prod,total,batch} = m_{prod,med,process} + m_{prod,md,process} + m_{prod,med,refilling} + m_{prod,md,refilling}`"
+   "Total operation time of one batch", ":math:`t_{batch,total} = t_{reflling} + N \times d_{t}`"
+   "System capacity associated to one MD module", ":math:`Cap_{module} = \frac{m_{prod,total,batch}}{t_{batch,total}}`"
+   "Total thermal power required", ":math:`Th_{total} = \Bigg(\frac{({\sum_{t=1}^{N}Th_{md,t}} + {\sum_{t=1}^{N}Th_{med,t}}) \times d_{t} + (Th_{md,t=0} + Th_{med,t=0}) \times t_{refilling}}{t_{batch,total}} \Bigg)`"
+   "Average STEC of a batch", ":math:`STEC = \frac{Th_{total} \times t_{batch,total}}{m_{prod,total,batch}}`"
```

Comment:
Can you add the Class Documentation section similar to example on documented models in watertap main? I am curious to see how your model_inputs config option shows up too.

Reply:
Sure. I added it but model_inputs didn't look good because all texts were wrapped up. So I removed it and referred to the table in the doc, please check the latest commit.